src/DynamoCoreWpf/UI/Themes/Modern/InPorts.xaml

```diff
@@ -200,6 +200,31 @@
                     CornerRadius="0,11,11,0"
                     IsHitTestVisible="True"
                     SnapsToDevicePixels="True" />
+
+            <Border x:Name="PortBorderHighlight"
+                    Grid.Column="1"
+                    Grid.ColumnSpan="7"
+                    BorderBrush="Purple"
+                    Height="29px"
+                    Margin="-7,0,0,0"
+                    BorderThickness="3,3,3,3"
+                    CornerRadius="0,11,11,0"
+                    Visibility="{Binding Highlight, UpdateSourceTrigger=PropertyChanged}"
+                    SnapsToDevicePixels="True">
+
+                <Border.Triggers>
```

Comment:
Not so sure about blinking. I'm leaning towards simple highlighting without blinking. What do you think ?

Reply:
I'm leaning towards simple highlighting but we can start from blinking and ask ppl to give it a try

Reply:
Well it looks like there is some issue with the animation specifically, it just stops working sometimes. It doesn't seem to be anything wrong anywhere. So I'll further look into it but for now I made the update to simple highlighting.